runtime/runtime-domain-service/ballerina/build.gradle

```diff
@@ -16,12 +16,37 @@
 // under the License.
 //
 
-plugins {
-    id 'net.researchgate.release' version '3.0.2'
-}
+import org.apache.tools.ant.taskdefs.condition.Os
 
 apply from: "$projectDir/../../../common-gradle-scripts/ballerina.gradle"
-
-tasks.named('build').configure {
+task build{
```

Comment:
When authoring tasks it's better follow [this format](https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:defining_tasks). So that it supports [Configuration Avoidance](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#task_configuration_avoidance).